core/arch/arm/plat-sam/conf.mk

```diff
@@ -42,3 +42,8 @@ CFG_TEE_RAM_VA_SIZE ?= 0x100000
 # Device tree related configuration
 CFG_DT_ADDR ?= 0x21500000
 CFG_GENERATE_DTB_OVERLAY ?= y
+
+CFG_WITH_SOFTWARE_PRNG ?= n
+CFG_ATMEL_TRNG ?= y
+CFG_HWRNG_PTA ?= y
+CFG_HWRNG_QUALITY ?= 1024
```

Comment:
`?=` ?
Atmel TRNG quality is not something configurable. I would rather use:
```
CFG_WITH_SOFTWARE_PRNG ?= n
CFG_ATMEL_TRNG ?= y
ifeq ($(CFG_ATMEL_TRNG),y)
CFG_HWRNG_PTA ?= y
$(call force,CFG_HWRNG_QUALITY,1024)
endif

Reply:
Indeed, looks better to force it.